modules/contact/doc/content/source/constraints/ComputeDynamicWeightedGapLMMechanicalContact.md

```diff
@@ -4,7 +4,7 @@
 
 This object is virtually analogous to [ComputeWeightedGapLMMechanicalContact](/ComputeWeightedGapLMMechanicalContact.md), but it is used for dynamic simulations.
 
-When the mortar mechanical contact constraints are used in dynamic simulations, the normal contact constraints need to be stabilized by ensuring that the normal gap time derivative is included to guarantee a dynamic contact. This is means of guaranteeing the 'persistency' condition, i.e. not only do we enforce the constraints instantaneously, but also that it will remain in contact. This approximate contact constraint stabilization is performed in  [ComputeDynamicWeightedGapLMMechanicalContact](/ComputeDynamicWeightedGapLMMechanicalContact.md). Once nodal contact is established, the constraint enforcement is switched to the persistency equation:
+When the mortar mechanical contact constraints are used in dynamic simulations, the normal contact constraints need to be stabilized by ensuring that the normal gap time derivative is included to guarantee a dynamic contact. This is means of guaranteeing the 'persistency' condition, i.e. not only do we enforce the constraints instantaneously, but also that the contact interface will remain in contact locally. This approximate contact constraint stabilization is performed in  [ComputeDynamicWeightedGapLMMechanicalContact](/ComputeDynamicWeightedGapLMMechanicalContact.md). Once nodal contact is established, the constraint enforcement is switched to the persistency equation:
```

Comment:
```suggestion
When the mortar mechanical contact constraints are used in dynamic simulations, the normal contact constraints need to be stabilized by ensuring that the normal gap time derivative is included to guarantee a dynamic contact. This is a means of guaranteeing the 'persistency' condition, i.e. not only do we enforce the constraints instantaneously, but also that the contact interface will remain in contact locally. This approximate contact constraint stabilization is performed in  `ComputeDynamicWeightedGapLMMechanicalContact`. Once nodal contact is established, the constraint enforcement is switched to the persistency equation:
```
I think this was linking back to itself if I'm reading it right?

Reply:
Reference to itself seemed acceptable to me, but I changed it.